examples/groth16-verifier/README.md

```diff
@@ -0,0 +1,29 @@
+# Groth16 Verifier Example
+
+This **experimental** example demonstrates the verification of a proof from another proof system, in this case Groth16 over the BN_254 curve (generated by Circom/SnarkJS), within the zkVM.
+
+## Summary
+This example uses a pre-generated `Groth16 proof` of a Sudoku solution generated via [zkSudoku].
+Specifically, you will learn:
+- How to verify a `Groth16 proof` generated by Circom/SnarkJS. To learn how to generate such a proof, follow the [zkSudoku] README or the Circom [tutorial]. You can also try with an [ECDSA-circuit] example created by [0xPARC].
+- How to prove the verification of a `Groth16 proof` from the zkVM.
```

Comment:
```suggestion
- How to prove the verification of a Groth16 proof in the zkVM.
```
Nit: "Groth16 proof" is not really a code literal, and so probably should not be in code format